.github/workflows/functional-test.yml

```diff
@@ -62,7 +64,9 @@ jobs:
     - uses: reactivecircus/android-emulator-runner@v2
       name: e2e_api${{ matrix.apiLevel }}
       with:
-        script: npm run e2e-test:driver
+        script: |
+          npm run e2e-test:driver
+          npm run e2e-test:commands
```

Comment:
Tried to have it [run in a different step ](https://github.com/appium/appium-uiautomator2-driver/pull/576/commits/d55d87e32747af34a7a00dfeee559c211ab2f31d) resulted in [a connection error to Appium server ](https://github.com/appium/appium-uiautomator2-driver/actions/runs/3968349318/jobs/6806217843)🤔 
Please let me know if you have better approaches to run multiple tests.